silnlp/nmt/config.py

```diff
@@ -413,14 +413,24 @@ def get_checkpoint_path(model_dir: Path, checkpoint_type: CheckpointType) -> Tup
     model_dir = SIL_NLP_ENV.get_source_experiment_path(model_dir)
     ckpt = None
     step = None
+    if type(checkpoint_type) is str:
```

Comment:
If `checkpoint_type` can be a `str`, then update the type to `Union[CheckpointType, str]`.

Reply:
Done.